platform-core-module/src/main/java/com/softicar/platform/core/module/program/AGProgram.java

```diff
@@ -24,14 +30,54 @@ public AGProgram setProgramUuid(UUID programUuid) {
 		return setProgramUuid(AGUuid.getOrCreate(programUuid));
 	}
 
+	public DayTime getQueuedAt() {
+
+		return getOrCreateProgramState().getQueuedAt();
+	}
+
+	public void saveQueuedAt(DayTime value) {
```

Comment:
This method is _only_ called from unit tests. It should be removed (test-only parts in production code should commonly be avoided). If necessary, equivalent methods can be added to the respective test classes.

Reply:
I changed that.